client/src/elements/molecules/item-list-entry.html

```diff
@@ -13,7 +13,7 @@
   </span>
 
   <delete-button
-    if.bind="user.roles.includes('poweruser')"
+    if.bind="user.roles.includes('poweruser') || user.id === item.conf.createdBy"
```

Comment:
This does not work as expected. `user.id` is not defined. This makes the user being able to delete items that do not have `createdBy` defined.

- We need a getter like for `user.roles` here https://github.com/nzzdev/Q-editor/blob/dev/client/src/resources/User.js#L53
- The property to check for is called `username`. So the getter should be named `username` and returning `this.data.username` like it's done in the roles getter.
- The check here should then check for equality against `user.username`

Reply:
I created a new property called username and compare this now with the createdBy-property of the item.
Deployed this in staging to test